railgun/test/integration/suite_test.go

```diff
@@ -75,7 +75,7 @@ var (
 	cluster ktfkind.Cluster
 
 	// watchNamespaces is a list of namespaces the controller watches
-	watchNamespaces = strings.Join([]string{elsewhere, corev1.NamespaceDefault}, ",")
+	watchNamespaces = strings.Join([]string{elsewhere, corev1.NamespaceDefault, "tcpingress"}, ",")
```

Comment:
Since this gets used in both the suite and TCPIngress test, it should be moved to a constant (similar to `elsewhere`) to avoid an inadvertent change making them out of sync.

Reply:
sure :+1:

Reply:
Sounds good, taken care of in e4f606c0bf3a529837fc25e507b81f05538cfc7d

Reply:
My bad, I shouldn't just say "this". To reword, the `watchNamespaces` string is fine, but it should be composed of constant namespace names, and tests should also use those names. So something like:

A constant declaration of `tcpIngressNamespace = "tcpingress"` like https://github.com/Kong/kubernetes-ingress-controller/blob/2f2a85707e785049e3f4c82750130a0dc9d4cb6e/railgun/test/integration/suite_test.go#L56-L57

```
watchNamespaces = strings.Join([]string{elsewhere, tcpIngressNamespace, corev1.NamespaceDefault}, ",")
```

And in tcpingress_test.go, `namespace := tcpIngressNamespace`

Reply:
Sure no worries, 0e94e619f8b367f0405d7be279e5f04f7b672e47 is the new patch for this :+1: